rotkehlchen/chain/gnosis/modules/monerium/constants.py

```diff
@@ -14,3 +15,10 @@
     string_to_evm_address('0x50D1A74F4b6dcaCddD97fd442C0e22a4c97F2b7f'),  # USDe
     string_to_evm_address('0x614Bd419D3735C9eb51542C06e5Acc09a9953f61'),  # ISKe
 }
+
+V1_TO_V2_MONERIUM_MAPPINGS: Final = {
```

Comment:
You have checked those right? That the mappings are all correct I mean

Reply:
Yes but I have double checked it